ansible/roles/prereq/templates/createImmortalDBs.sh.j2

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
Having this as a template makes the script dependent on ansible. Would it be a better way to just call a "normal" script with appropriate arguments?